README.md

```diff
@@ -27,7 +27,7 @@ http://docs.splunk.com/Documentation/Splunk/7.0.3/Data/UseHECusingconffiles
 Note the following when you generate your token:
 * Make sure that indexer acknowledgement is disabled for your token. 
 * Optionally, enable the indexer acknowledgement functionality by clicking the Enable indexer management checkbox. 
-* Do not generate your token using the default TLS cert provided by Splunk. For information about configuring Splunk to use self-signed or third-party certs, see http://docs.splunk.com/Documentation/Splunk/7.0.3/Security/AboutsecuringyourSplunkconfigurationwithSSL.
+* Do not generate your token using the default TLS cert provided by Splunk. The default certificates are not secure. For information about configuring Splunk to use self-signed or third-party certs, see http://docs.splunk.com/Documentation/Splunk/7.0.3/Security/AboutsecuringyourSplunkconfigurationwithSSL.
```

Comment:
can we have the latest release link instead of 7.0.3?

Reply:
Not all the docs are updated with 7.1 at release.  7.0.3 may be current for some time after Splunk Enterprise GA.